apidocs/removeNeighbors.md

```diff
@@ -0,0 +1,115 @@
+
+---
+### [removeNeighbors](https://github.com/iotaledger/iri/blob/dev/src/main/java/com/iota/iri/service/API.java#L573)
+ [AbstractResponse](/javadoc/com/iota/iri/service/dto/abstractresponse/) removeNeighborsStatement(java.util.List uris)
+
+Removes a list of neighbors to your node. 
```

Comment:
Removes a list of neighbors to your node. -> **Temporarily** removes a list of neighbors **from** your node.

This is only temporary, and if you have your neighbors added via the command line, they will be retained after you restart your node.
-> The added neighbors will be added again after relaunching IRI. Remove the neighbors from your config file or make sure you don't supply them in the `-n` command line option if you want to keep them removed after restart.